src/main/java/dev/minguinho/zeze/domain/slide/service/SlideService.java

```diff
@@ -53,21 +53,23 @@ public void update(Long slideId, SlideRequestDto slideRequestDto, Long userId) {
     }
 
     @Transactional
-    public void delete(Long slideId, Long userId) {
+    public void softDelete(Long slideId, Long userId) {
         Slide persist = findSlideIfAuthorized(slideId, userId);
-        slideRepository.delete(persist);
+        persist.delete();
+        slideRepository.save(persist);
     }
 
     private Page<Slide> getSlides(Long userId, PageRequest pageRequest) {
         if (Objects.isNull(userId)) {
-            return slideRepository.findAllByAccessLevel(Slide.AccessLevel.PUBLIC, pageRequest);
+            return slideRepository.findAllByAccessLevelAndDeletedAtIsNull(Slide.AccessLevel.PUBLIC, pageRequest);
         }
-        return slideRepository.findAllByUserIdOrderByUpdatedAtDesc(userId, pageRequest);
+        return slideRepository.findAllByUserIdAndDeletedAtIsNullOrderByUpdatedAtDesc(userId, pageRequest);
     }
 
     private Slide getSlide(Long slideId, Long userId) {
         Slide slide = slideRepository.findById(slideId)
             .orElseThrow(() -> new SlideNotFoundException(slideId));
```

Comment:
```suggestion
        Slide slide = slideRepository.findByIdAndDeletedAtIsNull(slideId)
            .orElseThrow(() -> new SlideNotFoundException(slideId));
```
로 통합하면 validateDeletedSlide가 필요 없어보여요!